src/Raven.Server/Web/Studio/LicenseHandler.cs

```diff
@@ -204,6 +207,74 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
                 }
             }
         }
+
+        [RavenAction("/studio/whats-new", "GET", AuthorizationStatus.ValidUser, EndpointType.Read)]
+        public async Task GetWhatsNewData()
+        {
+            if (ServerVersion.Build == ServerVersion.DevBuildNumber)
+            {
+                HttpContext.Response.StatusCode = (int)HttpStatusCode.BadRequest;
+                return;
+            }
+
+            var license = ServerStore.LoadLicense();
+
+            if (license == null)
+            {
+                HttpContext.Response.StatusCode = (int)HttpStatusCode.BadRequest;
+                return;
+            }
+            
+            var licenseId = license.Id.ToString();
+            
+            var request = new AboutViewRequest() { CurrentFullVersion = ServerVersion.FullVersion, LicenseId = licenseId };
+
+            var requestPayload = JsonConvert.SerializeObject(request);
+
+            var content = new StringContent(requestPayload, Encoding.UTF8, "application/json");
+
+            var upgradeInfoResponse = await ApiHttpClient.Instance.PostAsync("api/v1/upgrade/get-upgrade-info", content);
```

Comment:
Please note that API service can be temporary down, there might be connectivity issue etc

Reply:
Wrapped this part of code in `try catch` block, if we get an exception it's added to `ErrorMessage` property of `UpgradeInfoResponse`.